src/main/cpp/Robot.cpp

```diff
@@ -185,6 +230,7 @@ void Robot::TeleopPeriodic() {
             ),
             rc->limelight_util.m_swerve_drive_speeds[2]
```

Comment:
Just curious: What is the advantage of packing these related but distinct values into an array that obscures which value is which? What if you just break the array in `LimelightUtil` into individual members and name them accordingly? Then the code would read about like this:

```suggestion
            frc::Translation2d(
                units::meter_t(rc->limelight_util.m_swerve_drive_x_speed),
                units::meter_t(rc->limelight_util.m_swerve_drive_y_speed)
            ),
            rc->limelight_util.m_swerve_drive_rotation_speed
```